src/PointwiseFunctions/AnalyticSolutions/EinsteinSolutions/Minkowski.cpp

```diff
@@ -148,6 +149,7 @@ noexcept {
   EinsteinSolutions::Minkowski<DIM(data)>::extrinsic_curvature(                \
       const tnsr::I<DTYPE(data), DIM(data)>& x, double /*t*/) const noexcept;
 
+
```

Comment:
remove

Reply:
the blank line

Reply:
Removed

Reply:
Just FYI, whenever you push fixups we get emails, so if you're commenting to let us know you changed things you don't need to worry :)